src/dev/ci_setup/setup.sh

```diff
@@ -65,3 +65,8 @@ if [ "$GIT_CHANGES" ]; then
   echo -e "$GIT_CHANGES\n"
   exit 1
 fi
+
+###
+### copy .bazelrc for ci into $HOME/.bazelrc
+###
+cp "src/dev/ci_setup/.bazelrc" "$HOME/.bazelrc";
```

Comment:
Can we have the source file named `.bazelrc-ci` or something? Just to be more explicit and ensure it's not used.

Reply:
👍